.github/workflows/nvm.yml

```diff
@@ -5,9 +5,47 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: node --version
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - run: |
           nvm install --latest-npm --no-progress
-          echo "::set-env name=PATH::${NVM_BIN}:${PATH}"
+          echo "$(dirname $(which node))" >> $GITHUB_PATH
```

Comment:
if this should stay as adding `NVM_BIN` to the `$PATH`, happy to also make that change - just wasn't sure whether the `node` binary was also available at `NVM_BIN`

Reply:
Great catch! `NVM_BIN` is no longer necessary. 👍🏼